src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -98,7 +97,10 @@ def run(self, revision):
             '-p', 'obj-x86_64-pc-linux-gnu/',
             '-checks={}'.format(','.join(checks)),
         ] + modified_files
-        self.run_command(cmd, gecko_env=False)
+        clang_output = self.run_command(cmd, gecko_env=False)
+
+        # TODO Analyse clang output
+        logger.info('Clang output', output=clang_output)
 
     def run_command(self, cmd, gecko_env=True):
```

Comment:
Can we add a `run_gecko_command` or something like this? It could make sense to share the code, so that we sync changes between the three projects that use it.